angr/exploration_techniques/explorer.py

```diff
@@ -96,6 +96,7 @@ def setup(self, simgr):
 
     def step(self, simgr, stash='active', **kwargs):
         base_extra_stop_points = set(kwargs.get("extra_stop_points") or {})
+        kwargs.pop("extra_stop_points", None)
```

Comment:
This is kinda ugly. You can just change the above .get to a .pop with the empty set (you should probably change that dict literal to a set literal or maybe an empty tuple)